x/protorev/keeper/rebalance.go

```diff
@@ -30,13 +32,10 @@ func (k Keeper) IterateRoutes(ctx sdk.Context, routes []RouteMetaData, remaining
 
 		// If the profit is greater than zero, then we convert the profits to uosmo and compare profits in terms of uosmo
 		if profit.GT(sdk.ZeroInt()) {
-			if inputCoin.Denom != types.OsmosisDenomination {
-				uosmoProfit, err := k.ConvertProfits(ctx, inputCoin, profit)
-				if err != nil {
-					k.Logger(ctx).Error("Error converting profits: ", err)
-					continue
-				}
-				profit = uosmoProfit
+			profit, err := k.ConvertProfits(ctx, inputCoin, profit)
```

Comment:
I think we may also need to add bounding in ConvertProfits. If a CL pool is the highest liquidity pool then we will swap the entire amount without any tick bounds.

If a CL pool is the highest liquidity than I imagine this won't run into any issues, but maybe good to note in as a comment if we don't want to add the tick checking here that we are not doing so.

Reply:
adding a comment

Reply:
Am I correct in saying we went for the former and didn't just add a comment right?

Reply:
wait lol adding comment rn, misread the comment

Reply:
added the comment

Reply:
Per convo offline, we decided to not make the check. More likely than not we will not run into the issue of over-stepping `MaxTicksCrossed` in `ConvertProfits`. The upper bound check should be sufficient enough to cover most cases.